CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## x.y.z (unreleased)
 
 ### Added
-
+- Added setting for putting the media query style block to <body> vs <head>
```

Comment:
Could you please add the issue/PR number so this line is the same format as the other changelog entries?